### ISSUE TITLE
[WSL2] accessing physical disks

### ISSUE THREAD

**Issue by SvenDowideit** [microsoft/WSL#4207]
This is either a documentation issue, or a feature request :)

I have a dual boot (windows/linux) system - and have code that I work on for processing large datasets (my small test sets are in the TB range on a local ext4 ssd).

When I'm in Windows, I'd love to use WSL2 to keep working on this stuff.

So, if its "possible, or impossible/never going to happen" - can you point me at some documentation

if it needs dev work - 🥇  ?

**Comment by therealkenc** [comment]
What you are looking for is Hyper-V passthrough for WSL. Probably a [UserVoice](https://wpdev.uservoice.com/forums/266908-command-prompt-console-windows-subsystem-for-l) thing.

**Comment by PatrickLang** [comment]
+1 - I use a regular Hyper-V VM frequently with passthru disks for data recovery and conversion from other filesystems

**Comment by filmor** [comment]
The link to UserVoice refers back to this repository now ;-)

**Comment by ytrezq** [comment]
Seems the UserVoice is no longer available while the possibility to boot on a physical ext4 partition instead of a vhdx remains impossible.

**Comment by xzn** [comment]
This and the ability to have multiple vhdx attached to a disto would be very nice to have. Especially considering access to /mnt is currently very slow this would be a workaround for me.

**Comment by ytrezq** [comment]
@therealkenc : no ! I disagree. One day, the ability to use physical disks might be added but the ability to boot on it (which is the point of this issue) might still be missing (which is already the case for general HyperV).

Thus, I would rather say that that this issue depends on #689, but isn’t a duplicate.
